README.md

```diff
@@ -82,6 +82,7 @@ services:
       - env.list
     ports:
       - 5000:5000
+      - 5001:80
```

Comment:
Here we expose 5001:80, while in the docker-compose.yml we expose 8080:80 - why?

Reply:
It was a miss from my local development. I changed it to 5001 for consistency